src/Pages/Login/Login.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { SEND_TO_SERVER } from '../../config';
+
+export default function Login() {
+  const { Kakao } = window;
+  let history = useHistory();
+
+  const KakaoLoginClickHandler = () => {
+    Kakao.Auth.login({
+      success: function (authObj) {
+        sendTokenToServer();
+      },
+    });
+  };
+
+  function sendTokenToServer() {
```

Comment:
```suggestion
  const sendTokenToServer= () => {
```
일반 함수에서 this를 활용하는 경우가 아니라면 가급적 화살표 함수를 사용해주세요.

Reply:
네 바꾸겠습니다!